python/hsfs/util.py

```diff
@@ -139,6 +140,26 @@ def get_hudi_datestr_from_timestamp(timestamp):
     return datetime.fromtimestamp(timestamp / 1000).strftime("%Y%m%d%H%M%S%f")[:-3]
 
 
```

Comment:
I don't think we still actually need the `get_timestamp_from_date_string`. It would be nice if we could unify it with `convert_event_time_to_timestamp`.